scrapers/lobbyist/cli.py

```diff
@@ -0,0 +1,75 @@
+import functools
+
+import click
+
+
+def scrapelib_opts(f):
+    @click.option("--rpm", default=180, show_default=True)
+    @click.option("--retries", default=3, show_default=True)
+    @click.option("--verify/--no-verify", default=False, show_default=True)
+    @functools.wraps(f)
+    def wrapped_func(*args, **kwargs):
+        return f(*args, **kwargs)
+
+    return wrapped_func
```

Comment:
This was the main reason for pulling this into a CLI. That way, we can run scrapes faster locally by specifying --rpm=something larger than 3